sojo-api/app/models/Complex.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const mongodb = require ('@onehilltech/blueprint-mongodb')
+;
+
+var schema = new mongodb.Schema({
+  complex_name      : {type: String, required: true, trim: true},
+  landlord_email    : {type: String, required: true, trim: true},
+  maintenance_email : {type: String, required: true, trim: true},
+  address           : {type: String, required: true, trim: true}
```

Comment:
What does the field below represent?
```Node
address           : {type: String, required: true, trim: true}
```

Reply:
It is meant to be the address of the apartment complex. Would changing the name to complex_address help?